free-3.md

```diff
@@ -45,6 +91,23 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+Although tools like iTunes exists for general music listening and 
+[VLC](http://www.videolan.org/vlc/index.html) can
+provide streaming (both as host and client), no tool that I've been able
+to find enables these features in addition to allowing music from multiple
+sources. The closest I've been able to find is
+[Spotify](https://www.spotify.com/us/) which can allow users to listen to
+music streamed from Spotify or stored locally on the computer. For what
+I presume are legal reasons, however, you don't seem to be able to stream
+the files on your computer or incorperate YouTube sources. That being said,
+since I intend to only allow streaming to your own devices using either your
+own files, accounts, or publicably available content, this program does not
+suffer the same short comings as my Spotify client.
+
+Additionally, as mentioned before, [Foobar2000](https://www.foobar2000.org/)
+is an amazingly extendable media player that will likely be incorperated
+in my solution in some capacity (perhaps dealing with codecs?)
+
```

Comment:
I did some looking around and couldn't find anything either. Grooveshark used to do this, but then they got shut down for legal reasons.